lib/backpex/fields/has_many.ex

```diff
@@ -341,8 +341,7 @@ defmodule Backpex.Fields.HasMany do
     } = assigns
 
     link =
-      if link_assocs and Map.has_key?(field_options, :live_resource) and
-           LiveResource.can?(assigns, :show, item, live_resource) do
+      if link_assocs and Map.has_key?(field_options, :live_resource) and live_resource.can?(assigns, :show, item) do
```

Comment:
```suggestion
      if link_assocs and live_resource.can?(assigns, :show, item) do
```

The `live_resource` key is required so we can remove this condition. 

I guess, we also have to change this call: `live_resource.can?(assigns, :show, item)` and use the live resource of the association?

```suggestion
      if link_assocs and field_options.live_resource.can?(assigns, :show, item) do
```

(untested)